bin/v-import-directadmin

```diff
@@ -0,0 +1,311 @@
+#!/bin/bash
+# info: Import DirectAdmin backup to a new user
+#
+# example: v-import-directadmin /backup/backup.tar.gz
+#
+# Based on sk-da-importer
+# Credits: Maks Usmanov (skamasle) and contributors:
```

Comment:
```suggestion
# Credits: Maks Usmanov (skamasle), Jaap Marcus (jaapmarcus) and contributors:
```

Reply:
Fixed by amend commit, thanks for the feedback!